packages/framework/src/Model/Product/Search/Export/ProductSearchExportWithFilterRepository.php

```diff
@@ -163,8 +163,8 @@ protected function extractParameters(string $locale, Product $product): array
     protected function extractPrices(int $domainId, Product $product): array
     {
         $prices = [];
-        $productSellingPrices = $this->productFacade->getAllProductSellingPricesIndexedByDomainId($product)[$domainId];
-        /** @var \Shopsys\FrameworkBundle\Model\Product\Pricing\ProductSellingPrice $productSellingPrice */
+        $productSellingPrices = $this->productFacade->getAllProductSellingPricesByDomainId($product, $domainId);
```

Comment:
The `@var` phpdoc annotation should be always above the code it describes, sorry I didn't catch that the first time:
```suggestion
        /** @var \Shopsys\FrameworkBundle\Model\Product\Pricing\ProductSellingPrice[] $productSellingPrices */
        $productSellingPrices = $this->productFacade->getAllProductSellingPricesByDomainId($product, $domainId);
```